rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -56,14 +56,15 @@ class BakeRequest {
   String ami_name
   String ami_suffix
   Boolean upgrade
+  String instance_type
```

Comment:
I notice that the packer template treats this new attribute as required. It will take a bit of extra work in orca (the orchestration engine) to allow you to pass this new attribute from a pipeline bake stage all the way through to rosco. The reason for the extra work is that the internal Netflix bakery (which is not Rosco, but is largely interface-compatible with Rosco) will not accept requests with extraneous attributes. So the new attribute will need to be explicitly dropped from requests that are not routed to Rosco. I'll grab a link so you have it for later when you get to adding ui/orchestration support.